app/javascript/components/widgets/weather/moon-phase.jsx

```diff
@@ -155,7 +155,10 @@ MoonPhase.propTypes = {
   radius: PropTypes.string.isRequired,
   phase: PropTypes.string.isRequired,
   arcPath: PropTypes.string.isRequired,
-  arcAnimation: PropTypes.string.isRequired,
+  arcAnimation: PropTypes.shape({
```

Comment:
Changed Prop-Types entry for arcAnimation to hold two properties unique to the Keyframes model